CMakeLists.txt

```diff
@@ -68,3 +75,6 @@ endif ()
 #    DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/bin/hello_2
 #    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
 #)
+
+
+add_subdirectory(test)
```

Comment:
we already add_subdirectory(test), all you just need is to turn on the flag BUILD_TESTS when using cmake to generate the project build